site/content/docs/user/ingress.md

```diff
@@ -10,16 +10,8 @@ description: |-
 ---
 ## Setting Up An Ingress Controller
 
-We can leverage KIND's `extraPortMapping` config option when
-creating a cluster to forward ports from the host
-to an ingress controller running on a node.
-
-We can also setup a custom node label by using `node-labels`
-in the kubeadm `InitConfiguration`, to be used
-by the ingress controller `nodeSelector`.
-
-
 1. [Create a cluster](#create-cluster)
+2. Use [Cloud Provider KIND](./loadbalancer.md) to provision a Loadbalancer for the Ingress Controller.
```

Comment:
I think maybe I would phrase this as installing and running cloud provider kind to enable the loadbalancer controller which ingress-nginx will use through the loadbalancer API, unless we're going to do something where we actually have the user create a loadbalancer object, which we don't appear to be doing.


Ditto for how we describe it below.